patches/build.gen.py.patch

```diff
@@ -1,12 +1,41 @@
 diff --git a/build/gen.py b/build/gen.py
-index adb622a9..1b3558fe 100755
+index 232e5363..404239b9 100755
 --- a/build/gen.py
 +++ b/build/gen.py
-@@ -509,6 +509,7 @@ def WriteGNNinja(path, platform, host, options, args_list):
+@@ -201,8 +201,8 @@ def main(argv):
+                       default='../third_party/zoslib',
+                       dest='zoslib_dir',
+                       help=('Specify the path of ZOSLIB directory, to link ' +
+-                            'with <ZOSLIB_DIR>/install/lib/libzoslib.a, and ' +
+-                            'add -I<ZOSLIB_DIR>/install/include to the compile ' +
++                            'with <ZOSLIB_DIR>/lib/libzoslib.a, and ' +
++                            'add -I<ZOSLIB_DIR>/include to the compile ' +
+                             'commands. See README.md for details.'))
+ 
+   args_list.add_to_parser(parser)
+@@ -396,7 +396,7 @@ def WriteGNNinja(path, platform, host, options, args_list):
+       '.',
+   ]
+   if platform.is_zos():
+-    include_dirs += [ options.zoslib_dir + '/install/include' ]
++    include_dirs += [ options.zoslib_dir + '/include' ]
+ 
+   libs = []
```

Comment:
What's the reason for removing the 'install' dir from the above paths? As stated in [GN](https://gn.googlesource.com/gn/) doc, "On z/OS, building GN requires [ZOSLIB](https://github.com/ibmruntimes/zoslib) to be installed, as described at that URL. When building with build/gen.py, use the option --zoslib-dir to specify the path to [ZOSLIB](https://github.com/ibmruntimes/zoslib):".

If the zopen setup requires the install dir to be removed, then I suggest changing default to '../third_party/zoslib/install', and the latter part of the above statement to "use the option --zoslib-dir to specify the path to [ZOSLIB](https://github.com/ibmruntimes/zoslib) installation directory (that contains `include` and `lib` directories):".

Reply:
Hello John, I suggest reverting the above change, and buid zoslib using `build.sh -c -r -t` after https://github.com/ibmruntimes/zoslib/pull/41 has been merged (or you can clone its branch in the meantime); that should create the `install` directory under `zoslib/`.

Reply:
@netskink I just learnt the zopen branch of zoslib is no longer being maintained. You should be able to clone the **main** branch of zoslib and build it as described in my previous comment.